src/sqlfluff/core/dialects/dialect_bigquery.py

```diff
@@ -186,8 +182,6 @@ class SelectTargetElementSegment(AnsiSelectTargetElementSegment):
         type="function_name",
         _anti_template=r"STRUCT",
     ),
-    SelectTargetElementSegment=SelectTargetElementSegment,
```

Comment:
Why were we able to remove these?

Reply:
the `@bigquery_dialect.segment(replace=True)` decorator automatically adds it so that you don't also have to do this.